libethash-cl/CLMiner.cpp

```diff
@@ -342,32 +352,40 @@ void CLMiner::workLoop()
             }
 
             // Read results.
-			uint32_t count, gids[c_maxSearchResults];
+			SearchResults results;
```

Comment:
Please use spaces instead of tabs. Preferably (but not required) use `git-clang-format` to format all changes.

Reply:
I don't have git-clang-format...